### ISSUE TITLE
Edit roster in event-view

### ISSUE THREAD

**Issue by bslbckr** [ultical/ultical#26]
Currently, it is impossible to edit your team's roster if viewed in the event-view. You can expand your roster and view all your registered players but editing is impossbile although it looks exactly the same as it does in the team-view. For users this is kind of disturbing as they are assuming, they are doing anything wrong, in first place. 

Therefore, we should either allow to edit the roster in the event-view, too, or we should make it visually clear, that this view differs from the "editing-enabled-view" in the team-view.

**Comment by bastrapp** [comment]
Good point. This would be fairly easy to implement, my concern is as follows, though: As rosters are created not for one but potentially several events it might confuse the user if he changes a roster of event A that also the roster of event B changed.

A solution could be to create a dedicated team page (this will be necessary when team have more rosters and tournament attendances) and send the user to this page to edit the roster. Until then a short comment could clarify things.
